### DISCUSSION TITLE
Steps For Multiple Models

### DISCUSSION THREAD

**Discussion by dannydinges** [spatie/laravel-onboard#8] [Ideas]
I love the concept of this package but I feel like you should be able to set steps for multiple models. Maybe I missed something but I couldn't get it to work. I'd love to say these x steps are for ModelA and then these x are for ModelB. Is that possible currently?

**Comment by richardkeep** [comment]
Facing the same challenges. I want to Onboard a User and thereafter onboard their projects

**Comment by RhysLees** [comment]
Just made a PR for this https://github.com/spatie/laravel-onboard/pull/15

**Comment by RhysLees** [comment]
You can now do this in Release 2.4.0
